helpers/meetup_events.R

```diff
@@ -0,0 +1,87 @@
+###--------------------------------------------------------------------------###
+###   REMAINING STEPS:                                                       ###
+###   https://rladies.github.io/meetupr/articles/setup.html                  ###
+###--------------------------------------------------------------------------###
+# 1.  Save the MEETUPR_PWD environment variable as secret on CI (actually      #
+#     called repo secret for e.g. GitHub Actions)                              #
+# 2.  Use the code below to decrypt the secret before using meetupr. We save   #
+#     the temporary token to disk but to a temporary folder so it won’t end    #
+#     up in a package check artefact.                                          #
+###--------------------------------------------------------------------------###
+
+# Load packages
+library(dplyr)
```

Comment:
Unless there is a good reason these should generally be in lexicographical order. You do not have to change it, I am just giving a Pro tip

Reply:
Appreciate it John, and as is, there is not a reason for the order